a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -501,9 +501,9 @@ <h4>Recommended fields</h4>
 			<section id="additional-a11y-meta">
 				<h4>Additional accessibility metadata</h4>
 
-				<p>In this document are examples of important metadata that is expected to be in a wide range of
+				<p>In this document there are examples of important metadata that is expected to be in a wide range of
 					publications. However, we do not have examples of all possible features that the metadata can
-					express. The techniques have more values than are shown in the examples.</p>
+					express. 	more values than are shown in the examples.</p>
```

Comment:
new sentence here doesn't make sense without the start "The techniques have" has been remove.